PanModal/Presentable/PanModalPresentable+LayoutHelpers.swift

```diff
@@ -112,8 +112,11 @@ extension PanModalPresentable where Self: UIViewController {
 
         guard let application = UIApplication.value(forKeyPath: #keyPath(UIApplication.shared)) as? UIApplication
             else { return nil }
-
-        return application.keyWindow?.rootViewController
+        if #available(iOS 13.0, *) {
+            return application.windows.first?.rootViewController
+        }else {
+            return application.keyWindow?.rootViewController
+        }
```

Comment:
We can improve the code style this way:

```swift
if #available(iOS 13.0, *) {
    return application.windows.first?.rootViewController
}

return application.keyWindow?.rootViewController
```

Since the first statement already have a `return` so the `else` is not required